docs/specification.md

```diff
@@ -0,0 +1,155 @@
+# Homebase Lite Spec Draft
+
+This spec draft is for the proposed Tezos smart contract to be used as the on-chain component of a new lightweight Homebase tool.
+
+# Requirements
+
+This is intended to be a simpler option than baseDAO, both in complexity and usage, for communities to request feedback, run polls, etc.
```

Comment:
I suggest adding a link to baseDAO here (https://github.com/tezos-commons/baseDAO)